R/utils-json.R

```diff
@@ -193,7 +193,13 @@ getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, po
     for(i in seq_along(date_index_list)){
       d <- date_index_list[[i]]
       
-      applicable_dates <- points[['time']][d]
+      title <- getReportMetadata(data, 'title')
+      #result <- !objectName %in% listObjects
+      if (grepl(title, 'DV Hydrograph')) {
+        applicable_dates <- points[['time']][d] + hours(23) + minutes(59)
```

Comment:
so make the changes in dvhydrograph-data before it comes to getApprovals? I can try.